src/gui/Src/Gui/AssembleDialog.cpp

```diff
@@ -173,7 +173,7 @@ void AssembleDialog::on_lineEdit_textChanged(const QString & arg1)
 void AssembleDialog::on_checkBoxKeepSize_clicked(bool checked)
 {
     bKeepSizeChecked = checked;
-    mValidateThread->textChanged(ui->lineEdit->text());

```

Comment:
`ValidateExpressionThread::textChanged()` only takes action if `text` has changed. So this call to `textChanged()` has no effect.